BCC/src/main/java/BNU/presentation/MainView.java

```diff
@@ -101,6 +106,33 @@ public static void BuildLoginView(JFrame mainFrame, MainController controller) {
 		
 		//build center panel
 		
+		// auto close connection
+        try (Connection conn = DriverManager.getConnection(
+                "jdbc:postgresql://127.0.0.1:5432/BCC", "postgres", "aspartame")) {
```

Comment:
All of this code is going to be replaced by the DB wrapper, so we can remove it from here